xml/dm_registers.xml

```diff
@@ -153,8 +148,8 @@
             Debug Module after power up, including the platform's system reset
             or Debug Transport reset signals.
 
-            A debugger should pulse this bit low to ensure that the Debug
-            Module is fully reset and ready to use.
+            When first connecting, a debugger should pulse this bit low to
```

Comment:
I don't see a reason while Debugger A couldn't connect and put things in a certain state, then debugger B comes along and hooks up. There is no reason to *require* that B reset the state. Many debuggers may want to do this, but it shouldn't really be a requirement of the spec. 

The spec should state what the hardware and software agree on. Forcing the software to do certain things except what is required by the hardware shouldn't really be in the spec.